src/contribot/lib/contribot.py

```
@@ -0,0 +1,2046 @@
+            self._add_diagnosis(code="T0016", num_lines=len(self.df))
+            self.df = self.df[:10000]
+
+        self.df.index = self.df.index + 2
+        self.df = self.normalise_column_names(self.df)
+
+        # Create a column mapping for excel, the earlier version could not handle files with more than 26 cols (len(en-alphabet))
+        self.mapping = dict(
+            zip(
+                self.df.columns,
+                map_n_dataframe_cols_to_excel_cols(len(self.df.columns)),
+            )
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)